src/app.js

```diff
@@ -5,15 +5,10 @@ export default function (app) {
     return ""
   }
 
+  var emit = {event: event, hook: hook}
```

Comment:
`on` would be for registering events, we are no longer exposing an API for registering them. The current implementation registers everything in the `subscriptions` method.